integration-tests/cucumber-tests-platform-publiclighting/src/test/java/org/opensmartgridplatform/cucumber/platform/publiclighting/glue/steps/mocks/OslpDeviceSteps.java

```diff
@@ -106,12 +106,27 @@
      *            device.
      */
     @Then("^a get configuration \"([^\"]*)\" message is sent to device \"([^\"]*)\"$")
-    public void aGetConfigurationOSLPMessageIsSentToDevice(final String protocol, final String deviceIdentification) {
-        final Message message = this.oslpMockServer.waitForRequest(MessageType.GET_CONFIGURATION);
+    public void aGetConfigurationOSLPMessageIsSentToDevice(final String protocol, final String deviceIdentification)
+            throws DeviceSimulatorException {
+        this.aGetConfigurationOSLPMessageIsSentToSpecificDevice(protocol, deviceIdentification, this.getDeviceUID(new HashMap<>()));
+    }
+
+    /**
+     * Verify that a get configuration OSLP message is sent to the specific device.
+     *
+     * @param deviceIdentification
+     *            The device identification expected in the message to the
+     *            device.
+     */
+    @Then("^a get configuration \"([^\"]*)\" message is sent to device \"([^\"]*)\" with deviceUid \"([^\"]*)\"$")
+    public void aGetConfigurationOSLPMessageIsSentToSpecificDevice(final String protocol, final String deviceIdentification, final String deviceUID)
```

Comment:
As how things are build in our cucumber test set your statement is not true. (I am not saying that you are wrong, but that's not how the cucumber test are set up)

The default device_uid as set in PlatformDefaults.java = dGVzdDEyMzQ1Njc4. That decodes to test12345678
The default device_identifcation as set in PlatformDefaults.java = TD01

So in the tests device_uid and device_identifacation are not coupled in any way. If we do want to couple them, that will likely requires changes in many tests. I did not look into that. I also wa not aware of the (pretented) relatation between device_uid and device_identification. 

For now, I will suggest to keep the fields unrelated. Maybe it should be another issue on the back log.
What do you think?

Reply:
Agreed it would probably have too much impact to do something about the device identification vs UID split to make the tests simpler (if the assumptions hold up we can regard the relation between the two as fixed).